src/service/poc_lora.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+package helium.poc_lora;
+
+enum data_rate {
+  SF7BW125 = 0;
+  SF8BW125 = 1;
+  SF9BW125 = 2;
+  SF10BW125 = 3;
+  SF12BW125 = 4;
+  // TODO: add final values
+}
+
+enum invalid_reason {
+  bad_signature = 0;
+  // TODO: add final values
+}
+message lora_beacon_report_req_v1 {
+  bytes beacon_id = 1;  // likely be a concatenation of local and remote entropy so maybe not needed if we already include both fields ? 
+  bytes pub_key = 2;  
+  bytes local_entropy = 3;
+  bytes remote_entropy = 4;
+  bytes data = 5;
+  float frequency = 6;
+  int32 channel = 7;
+  data_rate datarate = 8;
+  int32 tx_power = 9;
+  // Timestamp of beacon report submit
+  uint64 timestamp = 10;
```

Comment:
UTC? Epoch?

Reply:
ideally this is, yeah, more precise than to the second